src/test/java/net/snowflake/client/core/SessionUtilLatestIT.java

```diff
@@ -465,4 +465,56 @@ public void testOktaAuthRetry() throws Throwable {
       SessionUtil.openSession(loginInput, connectionPropertiesMap, "ALL");
     }
   }
+
+  @Test
+  public void testOktaDisableSamlUrlCheck() throws Throwable {
+    SFLoginInput loginInput = createOktaLoginInput();
+    loginInput.setDisableSamlUrlCheck(true);
```

Comment:
Do we have a test where okta fails on step 5? if not - please add such

Reply:
I added two failed testing cases. Please have a look. Thank you.